framework/src/executor/tests/framework.rs

```diff
@@ -0,0 +1,283 @@
+use super::super::super::executor::ServiceExecutor;
+use async_trait::async_trait;
+use binding_macro::{cycles, service, tx_hook_after, tx_hook_before};
+use bytes::{Bytes, BytesMut};
+use cita_trie::MemoryDB;
+use protocol::traits::{
+    Context, Executor, ExecutorParams, ExecutorResp, Service, ServiceMapping, ServiceResponse,
+    ServiceSDK, Storage,
+};
+use protocol::types::{
+    Address, Block, Genesis, Hash, Proof, RawTransaction, Receipt, ServiceContext,
+    SignedTransaction, TransactionRequest,
+};
+use protocol::ProtocolResult;
+use std::sync::Arc;
```

Comment:
```suggestion
use std::sync::Arc;

use async_trait::async_trait;
use bytes::{Bytes, BytesMut};
use cita_trie::MemoryDB;

use super::super::super::executor::ServiceExecutor;

use binding_macro::{cycles, service, tx_hook_after, tx_hook_before};
use protocol::traits::{
    Context, Executor, ExecutorParams, ExecutorResp, Service, ServiceMapping, ServiceResponse,
    ServiceSDK, Storage,
};
use protocol::types::{
    Address, Block, Genesis, Hash, Proof, RawTransaction, Receipt, ServiceContext,
    SignedTransaction, TransactionRequest,
};
use protocol::ProtocolResult;
```

and the `super` keyword is disabled

Reply:
solved